src/App.tsx

```diff
@@ -1,27 +1,60 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import { get5First, getAll, getRedGoods } from './api/goods';
 
 // import { getAll, get5First, getRed } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const [allGoods, setAllGoods] = useState(false);
+  const [fiveGoods, setFiveGoods] = useState(false);
+  const [redGoods, setRedGoods] = useState(false);
```

Comment:
looks like redundant states

```suggestion
```
request should be on each click of the button
![Screenshot from 2023-08-04 13-15-29](https://github.com/mate-academy/react_dynamic-list-of-goods/assets/33029864/5361ee2e-70c0-499b-ab6f-67f4027f49df)